### PR TITLE
Dynatrace output plugin doc based on http output plugin

### PR DESCRIPTION
**Pull request by hans-lougas-dt**

### CODE REVIEW

**Comment by agardnerIT** [comment]
This is great. @robertjahn let me know about this.

One thing: In line with a "secure by design" mindset, I suggest we follow Fluent bit's recommendations in their official guide. On [this page](https://docs.fluentbit.io/manual/v/1.0/configuration/tls_ssl) they state that `tls.verify` is a setting to force certificate validation. The default is `on`.

They also state (on that page) that `(we strongly suggest always keep verification ON)`

**Comment by hans-lougas-dt** [comment]
Thanks @agardnerIT ! There's some discrepancy between this and Dynatrace documentation. I will sort it out

File: pipeline/outputs/dynatrace.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+| tls                        | Specify to use TLS                                                                                                                                                                                                                  | on                                                 |
+| tls.verify                 | TLS verification                                                                                                                                                                                                                    | off                                                |
+
+## Getting Started
+
+1. Get a [Dynatrace API](https://docs.dynatrace.com/docs/shortlink/api-authentication) token with the `logs.ingest` (Ingest Logs) scope.
+2. Determine your Dynatrace [environment ID](https://docs.dynatrace.com/docs/shortlink/monitoring-environment#environment-id).
+3. In your main Fluent Bit configuration file, append the following _Output_ section:
+
+```text
```
Comment (agup006) [inline pipeline/outputs/dynatrace.md:33]: Could we also add a yaml sample?

**Review by agup006 — APPROVED** [review]
Looks great! Will be nice to have this in docs

**Comment by hans-lougas-dt** [comment]
@agup006 thanks, do you think the current version would also fit already for FB doc?
before we add yaml sample

**Comment by edsiper** [comment]
Hi, since this aims to list a Vendor which uses a generic built-in plugin, I would like to propose to open the discussion to find a different page/section in our documentation to fill this need: as a bit of context the `outputs` in our documentation list the plugins available, but not vendor destinations.

side note, today I was informed another company wanted to accomplish something similar where they offer a hosted Loki endpoint and basically looking to be promoted as one of the supported services by using the company name. 

ideas are welcome :)

**Comment by agup006** [comment]
I would say let’s merge this then rethink structure for all plugins. We already have some pages that are derived from standard plugins.

An ideal way would be to keep separate page but as a subpage of an existing plugin 

e.g.

Plugins > Destinations > HTTP which has sub bullets such as 

* Dynatrace
* Observe

**Pull request merged**